### PR TITLE
343 compare optimized flh with previous dataset

### PR DESCRIPTION
**Pull request by markushal**

### CODE REVIEW

File: tests/test_opt.py [hunk 1]
```diff
@@ -166,3 +166,191 @@ def test_e_cyclic_period_minimal_example():
...
+@pytest.fixture()
+def flh_data_new(api) -> pd.DataFrame:
+    """Load optimization results from csv files.
+
+    and prepare them for comparison with old dataset.
+    You need to download them from the server first:
+
+    ````
+    scp ptxboa2:/home/ptxboa/ptx-boa_offline_optimization/optimization_cache/*.csv .
+    ````
```
Comment (markushal) [inline tests/test_opt.py:248]: @joAschauer You need to download the data from the server first

**Comment by markushal** [comment]
@joAschauer what could also be interesting (and maybe simpler to implement): Once the app can access the optimization results, we can just export the cost calculation results from the API for two versions of the app (before and after using the optimization results), and compare them. 

that way, we could compare two datasets that have the same dimensions. 

We could even add a parameter to the api that makes using the optimization results optional, then we would not have compare two versions of the app

**Comment by joAschauer** [comment]
> @joAschauer what could also be interesting (and maybe simpler to implement): Once the app can access the optimization results, we can just export the cost calculation results from the API for two versions of the app (before and after using the optimization results), and compare them.

Yes, we can try to do that. However, in that case we can not directly compare the full load hours.

**Comment by markushal** [comment]
@joAschauer this is the notebook we discussed today.

**Pull request closed without merging**